### DISCUSSION TITLE
Projekt noch aktuell?

### DISCUSSION THREAD

**Discussion by flatty2804** [Trawacho/StockTV#7] [General]
Hallo, 

ist das Projekt noch aktuell bzw. wird es noch weiter entwickelt oder supported?
Ich bin schon sehr interessiert...kann man dich auch per Email oder ähnliches mal kontaktieren?

Danke
Grüße Arthur

**Comment by bachmichl** [comment]
Servus Arthur,
jupp, das Projekt läuft. Daniel betreut/entwickelt das Ganze: stocktv@gmx.de
Servus,
Mich

**Comment by Trawacho** [comment]
closed
